cmd/follow.go

```diff
@@ -21,6 +21,8 @@ import (
 	"github.com/spf13/cobra"
 )
 
+const followEnabled = false
```

Comment:
Should be enabled. Kjell Erik suggested that we try to deliver on the issue described here, as well as deliver on deploy-only:
https://github.com/equinor/radix-platform/issues/225